internal/planner/configuration.go

```diff
@@ -81,11 +81,11 @@ func (pl *Planner) idmapOptions() smbcc.SmbOptions {
 // Update the held configuration based on the state of the instance
 // configuration.
 func (pl *Planner) Update() (changed bool, err error) {
-	globals, found := pl.ConfigState.Globals[smbcc.Globals]
+	_, found := pl.ConfigState.Globals[smbcc.Globals]
 	if !found {
 		globalOptions := smbcc.NewGlobalOptions()
 		globalOptions.SmbPort = pl.GlobalConfig.SmbdPort
-		globals = smbcc.NewGlobals(globalOptions)
+		globals := smbcc.NewGlobals(globalOptions)
```

Comment:
This is interesting..

Above change was already merged as part of #247 and GitHub couldn't find any conflict for the current PR?

Reply:
I just rebased. git didn't prompt me to resolve conflicts either, so my guess is that both cli git and github determined it was not a conflict that needed manual resolution. Regardless, the rebase has now eliminated that difference.